centreon/tests/e2e/features/Additional-connectors/01-ACC-creation.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Create a new Additional Connector Configuration
+  As a Centreon user
+  I want to visit the Additional Connector Configuration page
+  To manage additional connector configuration
+
```

Comment:
You need to create a test case (TC) for each scenario and ensure that the corresponding tags are added, following the same approach as in the other features